internal/testutils/trace_pipeline_builder.go

```diff
@@ -32,19 +32,15 @@ func (b *TracePipelineBuilder) WithName(name string) *TracePipelineBuilder {
 	return b
 }
 
-func (b *TracePipelineBuilder) WithEndpoint(endpoint string) *TracePipelineBuilder {
-	b.endpoint = endpoint
-	return b
-}
-
-func (b *TracePipelineBuilder) WithBasicAuth(user, password string) *TracePipelineBuilder {
-	b.basicAuthUser = user
-	b.basicAuthPassword = password
+func (b *TracePipelineBuilder) WithStatusCondition(cond metav1.Condition) *TracePipelineBuilder {
+	b.statusConditions = append(b.statusConditions, cond)
 	return b
 }
 
-func (b *TracePipelineBuilder) WithStatusCondition(cond metav1.Condition) *TracePipelineBuilder {
-	b.statusConditions = append(b.statusConditions, cond)
+func (b *TracePipelineBuilder) OtlpOutput(opts ...OTLPOutputOption) *TracePipelineBuilder {
```

Comment:
In the opts you are using OTLP, but here it's Otlp. Maybe let's unify it and use OTLP everywhere?

Reply:
done